### PR TITLE
[docs] Update the README

### PR DESCRIPTION
**Pull request by danilo-leal**
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/mui/material-ui/blob/HEAD/CONTRIBUTING.md#sending-a-pull-request).

Adapting the README with what I found relevant for Base UI specifically. Will comment on some things in particular.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,33 @@
...
+[![npm downloads](https://img.shields.io/npm/dm/@mui/material.svg)](https://www.npmjs.com/package/@mui/base)
+[![CircleCI](https://circleci.com/gh/mui/material-ui/tree/master.svg?style=shield)](https://app.circleci.com/pipelines/github/mui/base-ui?branch=master)
+[![Coverage Status](https://img.shields.io/codecov/c/github/mui/material-ui/master.svg)](https://codecov.io/gh/mui/base-ui/branch/master)
 [![Follow on X](https://img.shields.io/twitter/follow/MUI_hq.svg?label=follow+MUI)](https://twitter.com/MUI_hq)
-[![Renovate status](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://github.com/mui/material-ui/issues/27062)
-[![Average time to resolve an issue](https://isitmaintained.com/badge/resolution/mui/material-ui.svg)](https://isitmaintained.com/project/mui/material-ui 'Average time to resolve an issue')
+
+<!-- [![Renovate status](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://github.com/mui/material-ui/issues/27062) -->
+<!-- [![Average time to resolve an issue](https://isitmaintained.com/badge/resolution/mui/material-ui.svg)](https://isitmaintained.com/project/mui/material-ui 'Average time to resolve an issue') -->
```
Comment (danilo-leal) [inline README.md:18]: 1. Do we have a similar issue for the Renovate status to link in Base UI's case?
2. Don't think we have/need this at the moment, right?

File: README.md [hunk 2]
```diff
@@ -1,71 +1,33 @@
...
 Visit [https://mui.com/base-ui/](https://mui.com/base-ui/) to view the full documentation.
 
-**Note**: Base UI is still in beta.
-We are adding new components regularly and you're welcome to contribute!
-
-### MUI System
-
-Visit [https://mui.com/system/getting-started/](https://mui.com/system/getting-started/) to view the full documentation.
+**Note**: Base UI is currently being rewritten and isn't adding new features or components to the current implementation.
+More details about future plans for it can be found in this blog post.
```
Comment (danilo-leal) [inline README.md:30]: This callout (with the revised wording) will also be available in the documentation like we talked 👌

File: README.md [hunk 3]
```diff
@@ -104,28 +66,17 @@ See the full list of [our backers](https://mui.com/material-ui/discover-more/bac
...
 
 Read the [contributing guide](/CONTRIBUTING.md) to learn about our development process, how to propose bug fixes and improvements, and how to build and test your changes.
 
-Contributing is about more than just issues and pull requests!
-There are many other ways to [support MUI](https://mui.com/material-ui/getting-started/faq/#mui-is-awesome-how-can-i-support-the-project) beyond contributing to the code base.
-
 ## Changelog
 
-The [changelog](https://github.com/mui/material-ui/releases) is regularly updated to reflect what's changed in each new release.
+The [changelog](https://github.com/mui/base-ui/releases) is regularly updated to reflect what's changed in each new release.
```
Comment (danilo-leal) [inline README.md:75]: Maybe we should update the wording here, given there won't be any "regular updates" for a few months until v1?

File: README.md [hunk 4]
```diff
@@ -104,28 +66,17 @@ See the full list of [our backers](https://mui.com/material-ui/discover-more/bac
...
 
 ## Roadmap
 
-Future plans and high-priority features and enhancements can be found in our [roadmap](https://mui.com/material-ui/discover-more/roadmap/).
+Future plans and high-priority features and enhancements can be found in [our roadmap](https://mui.com/base-ui/getting-started/roadmap/).
```
Comment (danilo-leal) [inline README.md:79]: I'll add this page for the Base UI docs now that we made all GitHub projects public!

File: README.md (hunk 3 above)
Comment (colmtuite) [reply README.md:75]: Perhaps "The [changelog](https://github.com/mui/base-ui/releases) reflects what has changed in each new release."

**Review by colmtuite — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (oliviertassinari) [reply README.md:18]:
```suggestion
[![Renovate status](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://github.com/mui/base-ui/issues/2)
<!-- [![Average time to resolve an issue](https://isitmaintained.com/badge/resolution/mui/material-ui.svg)](https://isitmaintained.com/project/mui/material-ui 'Average time to resolve an issue') -->
```

File: README.md (hunk 4 above)
Comment (michaldudak) [reply README.md:78]: Can we link the GH project here instead?

Comment (danilo-leal) [reply README.md:78]: I think that makes sense, however, I was planning to add a similar Roadmap page to the Base docs as we have on the Joy UI docs, which includes a little bit of rationale about the roadmap (and it then links to the GitHub project).

**Comment by danilo-leal** [comment]
I have no idea why those checks are failing 🤔 — Update: more info on [this Slack thread](https://mui-org.slack.com/archives/C042YB5RB3N/p1709629350987879).

**Pull request merged**